src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -128,7 +133,8 @@ public static class EditPersonDescriptor {
         private Name name;
         private Phone phone;
         private Email email;
-        private Address address;
+        private Telegram handle;
+        private GitHub username;
```

Comment:
Should we standardise the `GitHub` field to be named as either `username` or `github`? 

Because in `Person.java` it is named as `github` but here it is named as `username`.